pkg/controller/namespace/namespace_controller_utils.go

```diff
@@ -184,6 +184,12 @@ func deleteAllContent(kubeClient clientset.Interface, versions *unversioned.APIV
 				return estimate, err
 			}
 		}
+		if containsResource(resources, "replicasets") {
```

Comment:
Are you referring to the test case or when running the actual code?

The test case is mocked up, so you need to add your resource:
https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/namespace/namespace_controller_test.go#L151